template.rb

```diff
@@ -63,6 +63,11 @@ def add_template_repository_to_source_path
 say 'Copying circleci configuration'
 directory 'lib/install/.circleci', '.circleci'
 
+say '-------------------------------------------------------------'
+say 'For running auto-update of Gemfile.tools with CircleCI,'
+say 'please, provide GITHUB_TOKEN environment variable in CircleCI'
```

Comment:
also pronto github key ;) we could even use one key what do you think?

Reply:
@pftg Where do we use it?

Reply:
https://github.com/jetthoughts/jt_tools/blob/master/lib/install/.circleci/config.yml#L120

Reply:
@pftg Is it what you mean https://github.com/jetthoughts/jt_tools/pull/5/commits/4c5c0f4a366a8f636f25f98f65ffb5cc4fd9d659 ?